commit 80b767438f16c21482b40a4028317988b623e652
Author: Derek Mauro <dmauro@google.com>
Date:   Mon May 12 13:43:47 2025 -0700

    Remove relocatability test that is no longer useful
    
    PiperOrigin-RevId: 757895298
    Change-Id: Id0a981b11499abc1471fe70589cfd5dfa13a9c2b

diff --git a/absl/container/flat_hash_map_test.cc b/absl/container/flat_hash_map_test.cc
index 5c83c941..e1d9382a 100644
--- a/absl/container/flat_hash_map_test.cc
+++ b/absl/container/flat_hash_map_test.cc
@@ -116,15 +116,6 @@ TEST(FlatHashMap, StandardLayout) {
 
 TEST(FlatHashMap, Relocatability) {
   static_assert(absl::is_trivially_relocatable<int>::value);
-#if ABSL_INTERNAL_CPLUSPLUS_LANG <= 202002L
-  // std::pair is not trivially copyable in C++23 in some standard
-  // library versions.
-  // See https://github.com/llvm/llvm-project/pull/95444 for instance.
-  // container_memory.h contains a workaround so what really matters
-  // is the transfer test below.
-  static_assert(
-      absl::is_trivially_relocatable<std::pair<const int, int>>::value);
-#endif
   static_assert(
       std::is_same<decltype(absl::container_internal::FlatHashMapPolicy<
                             int, int>::transfer<std::allocator<char>>(nullptr,
